### DISCUSSION TITLE
Application Status of Github Campus Expert | Aug 2022

### DISCUSSION THREAD

**Discussion by singhmansi25** [community/community#34205] [GitHub Education]
I have applied for Github Campus Expert in August batch of 2022. But I haven't received any mail regarding application status yet. When can I expect the mail or whom can I contact?
The website for application status is not working as well.

**Comment by juanpflores** [comment]
I'm pretty sure we have already sent all the emails to applicants. Did you apply using this username?

**Reply by singhmansi25** [reply]
Yes, I applied using this username only, but I haven't received any mail yet.
Please guide.

**Reply by singhmansi25** [reply]
@juanpflores

**Comment by cosmicqbit** [comment]
You should be able to check the status of your application at:
https://apply.githubcampus.expert/status

**Reply by singhmansi25** [reply]
@cosmicqbit 
I have tried checking status using this link, but it redirects me to [https://apply.githubcampus.expert/subscribe](https://apply.githubcampus.expert/subscribe) the subscribe page.
I haven't received any mail yet.

**Reply by singhmansi25** [reply]
Is there any support mail, where I can address my issue?

**Reply by singhmansi25** [reply]
@juanpflores 
@cosmicqbit

**Comment by cosmicqbit** [comment]
The deadline to submit the video clip was 2nd October. Github started sending out responses from 19 Oct, so it is already over. However I know many students who didn't received any mail from github campus at all, which basically is cuz of bug in their automated mailing system. Try campusexperts@github.com

**Reply by cosmicqbit** [reply]
Also [check this discussion](https://github.com/orgs/community/discussions/34984)
